### PR TITLE
Add "publish to Artifact Hub" page

### PR DESCRIPTION
**Pull request by flavio**
Explain how anybody can publish their own policies to Artifact Hub.

Part of https://github.com/kubewarden/policy-hub/issues/190 epic.

### CODE REVIEW

File: docs/distributing-policies/publish-policy-to-artifact-hub.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Then press the *"Add"* button and fill the form:
+
+  * Choose *"Kubewarden policies"* as kind.
+  * Enter a *"Name"* and *"Display name"* of your choice.
+  * Enter the URL to your Git repository.
+
+Finally, press the *"Add"* button. This will bring you back to the *"Repositories"*
+page, where you will see your freshly created repository.
+
+Each repository has a series of information. Find the *"ID"* property of the
```
Comment (viccuad) [inline docs/distributing-policies/publish-policy-to-artifact-hub.md:83]: Nit:

```suggestion
Each repository has several information fields. Find the *"ID"* property of the
```

Or something similar, it's just that here the plural + singular of `series of information` sounds weird.

**Review by viccuad — APPROVED** [review]
LGTM! Nice writeup!

**Review by jvanz — APPROVED** [review]

**Review by jvanz — APPROVED** [review]
The text LGTM. The only suggestion that I have is to add screen shoots of the pages mentioned in the text.

File: docs/distributing-policies/publish-policy-to-artifact-hub.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+install and publish packages and configurations for CNCF projects.
+
+Kubewarden policies can be published on Artifact Hub and made
+discoverable to the wide audience of CNCF users.
+
+:::note
+Artifact Hub is content aggregator, it doesn't actually host the
+artifacts that are published on it.
+
+Kubewarden Policies need to be physically hosted by a container
```
Comment (divya-mohan0209) [inline docs/distributing-policies/publish-policy-to-artifact-hub.md:18]: So this is a general requirement for Artifact Hub and I think should be stated as such.
```suggestion
Artifact Hub requires you to physically host container image repositories on a container
```

File: docs/distributing-policies/publish-policy-to-artifact-hub.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  will be visible on Artifact Hub.
+
+The last approach is what is used inside of our official policy templates.
+The Git repository that is scaffolded will include the `artifacthub-pkg.yml`
+and `artifacthub-repo.yml` files.
+
+## Publishing Steps
+
+Before publishing a policy to Artifact Hub, you must create an account on
+the [website](https://artifacthub.io/). To do that, use the *"Sign Up"* button
```
Comment (divya-mohan0209) [inline docs/distributing-policies/publish-policy-to-artifact-hub.md:52]: Just mentioning this should suffice. We do not want to explicitly provide click-this-button type steps which should be common knowledge. However, providing the context that signup is mandatory is relevant.
```suggestion
the [website](https://artifacthub.io/).
```

File: docs/distributing-policies/publish-policy-to-artifact-hub.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+:::note
+Right now the contents of the `artifacthub-repo.yml` file are not relevant.
+:::
+
+Once everything is in place, log into Artifact Hub and go to your
+[control plane](https://artifacthub.io/control-panel/repositories?page=1).
+
+Decide whether you want to publish the policy as an user or under an Artifact Hub
+organization you belong to. This is done by choosing the right *"control panel context"*.
```
Comment (divya-mohan0209) [inline docs/distributing-policies/publish-policy-to-artifact-hub.md:72]:
```suggestion
organization you belong to. This is done by choosing the correct *"control panel context"*.
```

**Review by divya-mohan0209 — COMMENTED** [review]
Rephrased & added a few comments. Hopefully they're helpful!

File: docs/distributing-policies/publish-policy-to-artifact-hub.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+:::note
+Artifact Hub is content aggregator, it doesn't actually host the
+artifacts that are published on it.
+
+Kubewarden Policies need to be physically hosted by a container
+registry or a web server.
+See the *"[distributing policies](../distributing-policies)"* section for more information about how to
+host your policies.
+:::
+
```
[the remark this answers is not in the document]
Comment (flavio) [reply docs/distributing-policies/publish-policy-to-artifact-hub.md:23]: Sorry, I don't get this comment.

The next section is going to talk about how to do publishing on Artifact Hub. But, to publish on Artifact Hub you still need to have your policy hosted somewhere. So this "note" section is there to clarify that Artifact Hub is **not** meant as a replacement for a container registry (let's say `ghcr.io`); you still need to find a hosting solution for your `.wasm` file

**Comment by flavio** [comment]
> The text LGTM. The only suggestion that I have is to add screen shoots of the pages mentioned in the text.

I would prefer to avoid doing that, because the UI of Artifact Hub could change breaking our docs.

File: docs/distributing-policies/publish-policy-to-artifact-hub.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  * Choose *"Kubewarden policies"* as kind.
+  * Enter a *"Name"* and *"Display name"* of your choice.
+  * Enter the URL to your Git repository.
+
+Finally, press the *"Add"* button. This will bring you back to the *"Repositories"*
+page, where you will see your freshly created repository.
+
+Each repository has several information fields. Find the *"ID"* property of the
+repository you just created and copy it.
```
Comment (flavio) [inline docs/distributing-policies/publish-policy-to-artifact-hub.md:83]: @divya-mohan0209 : thought about what you said during the standup. What about making everything less "artifact hub docs":

```suggestion
From your control plane, add a new Artifact Hub by following the guided procedure.
Once this done, copy the value of the *"ID"* property that Artifact Hub
assigned to the freshly created repository.
```

Comment (divya-mohan0209) [reply docs/distributing-policies/publish-policy-to-artifact-hub.md:83]: I think this looks good, Flavio. However, if the guided procedure doesn't include the actual steps you mentioned Id rather we retain it as-is.

File: docs/distributing-policies/publish-policy-to-artifact-hub.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (divya-mohan0209) [reply docs/distributing-policies/publish-policy-to-artifact-hub.md:23]: So - what I meant was after the note & before the actual heading below, we need to add a statement that we're going to talk about the publishing process. Just as a segue.

**Review by divya-mohan0209 — COMMENTED** [review]
Left a comment, but overall LGTM

File: docs/distributing-policies/publish-policy-to-artifact-hub.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (flavio) [reply docs/distributing-policies/publish-policy-to-artifact-hub.md:23]: Understood. I've fixed that

File: docs/distributing-policies/publish-policy-to-artifact-hub.md (hunk 6 above)
Comment (flavio) [reply docs/distributing-policies/publish-policy-to-artifact-hub.md:83]: Currently ArtifactHub's docs do not have this kind of content. Hence I'll leave this text unchanged.

Comment (divya-mohan0209) [reply docs/distributing-policies/publish-policy-to-artifact-hub.md:86]: That is fine by me!

**Pull request merged**